cli/flags.rs

```diff
@@ -1487,6 +1511,40 @@ fn ca_file_arg_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
   flags.ca_file = matches.value_of("cert").map(ToOwned::to_owned);
 }
 
+fn location_arg_parse(
+  flags: &mut Flags,
+  matches: &clap::ArgMatches,
+) -> clap::Result<()> {
+  flags.location = match matches.value_of("location") {
+    None => None,
+    Some(href) => Some(
+      Url::parse(href)
+        .map_err(|e| e.to_string())
+        .and_then(|url| {
+          if !["http", "https", "file"].contains(&url.scheme()) {
```

Comment:
I think the most recent consensus was that we shouldn't allow `file://` for the first pass, because it does not work with relative fetch. Can you remove this protocol from the supported list for now?